gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
-|   |  | | * **0** or **(empty)** - Frequency-based trips are not exactly scheduled. This is the default behavior. |
-|   |  | | * **1** - Frequency-based trips are exactly scheduled. For a frequencies.txt row, trips are scheduled starting with trip_start_time = start_time + x * headway_secs for all x in (0, 1, 2, ...) where trip_start_time < end_time. |
-|   |  | | The value of **exact_times** must be the same for all frequencies.txt rows with the same **trip_id**. If **exact_times** is 1 and a frequencies.txt row has a **start_time** equal to **end_time**, no trip must be scheduled. When **exact_times** is 1, care must be taken to choose an **end_time** value that is greater than the last desired trip start time but less than the last desired trip start time + **headway_secs**. |
+|  trip_id | ID referencing `trips.trip_id` | **Required** | Identifies a trip to which the specified frequency of service applies. |
+|  start_time | Time | **Required** | Time at which the first vehicle departs from the first stop of the trip with the specified frequency. |
+|  end_time | Time | **Required** | Time at which service changes to a different frequency (or ceases) at the first stop in the trip. |
+|  headway_secs | Non-negative integer | **Required** | Time, in seconds, between departures from the same stop (headway) for the trip, during the time interval specified by `start_time` and `end_time`. Multiple headways for the same trip are allowed, but may not overlap. New headways may start at the exact time the previous headway ends.  |
+|  exact_times | Enum | Optional | Indicates whether frequency-based trips should be exactly scheduled based on the specified headway information. The value of `exact_times` must be the same for all [frequencies.txt](#frequenciestxt) records with the same `trip_id`. If `exact_times`=`1` and a [frequencies.txt](#frequenciestxt) record has a `start_time` equal to `end_time`, no trip must be scheduled. When `exact_times`=`1`, care must be taken to choose an `end_time` value that is greater than the last desired trip `start_time` but less than the last desired trip start\_time + headway\_secs. Valid values are: |
+|   |  | | `0` or "empty" - Frequency-based trips are not exactly scheduled.|
+|   |  | | `1` - Frequency-based trips are exactly scheduled with with trip\_start\_time = start\_time + x * headway\_secs for all x in (0, 1, 2, ...) where trip\_start\_time < end\_time. |
 
 ### transfers.txt
 
 File: **Optional**
 
-Trip planners normally calculate transfer points based on the relative proximity of stops in each route. For potentially ambiguous stop pairs, or transfers where you want to specify a particular choice, use transfers.txt to define additional rules for making connections between routes.
+GTFS consumers calculate transfer points based on allowable time and stop proximity. [Transfers.txt](#transferstxt) specifies additional rules and overrides for selected transfers. 
```

Comment:
Proposed change: "GTFS-consuming applications identify available transfers in passenger journeys based on allowable time and stop proximity."

What do you think of that? I'm not sure but I think it might be useful to introduce the notion of a passenger journey.

Reply:
I'm not sure that we need to specify that transfers occur during rider journeys. Is there anywhere else a transfer could occur?